bumble/device.py

```diff
@@ -778,6 +779,7 @@ def __init__(self) -> None:
         self.irk = bytes(16)  # This really must be changed for any level of security
         self.keystore = None
         self.gatt_services: List[Dict[str, Any]] = []
+        self.address_resolution = False
```

Comment:
It might be better to call this `address_resolution_in_controller` or `address_resolution_offload`, because address resolution always happens, it's just whether it happens on the host or the controller that's different.

Reply:
Done